scripts/docs_version_pragma_check.sh

```diff
@@ -32,13 +32,114 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+function getAllAvailableVersions()
+{
+    allVersions=()
```

Comment:
I actually think this can be shortened to
```
allVersions=$(wget -q -O- https://ethereum.github.io/solc-bin/bin/list.txt | grep -Po '(?<=soljson-v)\d+.\d+.\d+(?=\+commit)')
```

Reply:
Ok, you don't get the filter about breaking at 0.4.11.

Reply:
Oh and you could add: `| sort  -V ` to sort by version.